package.json

```diff
@@ -27,16 +28,20 @@
     "@types/express": "^4.0.37",
     "@types/express-session": "^1.15.3",
     "@types/nunjucks": "^3.1.3",
+    "@types/redis": "^2.8.14",
     "del": "^5.1.0",
-    "gulp": "^4.0.0",
+    "inversify": "^5.0.1",
+    "inversify-express-utils": "^6.3.2",
+    "gulp": "^4.0.2",
     "gulp-sass": "^4.0.2",
     "gulp-typescript": "^6.0.0-alpha.1",
     "gulp-util": "^3.0.8",
     "gulp-watch-sass": "^1.4.1",
     "node-sass": "^4.13.1",
     "sass-lint": "^1.13.1",
+    "reflect-metadata": "^0.1.13",
```

Comment:
That is more runtime dependency, lack of it will stop an app from starting. I suggest you move it to a block above.